CHANGES.rst

```diff
@@ -148,6 +148,7 @@ astropy.visualization
 astropy.wcs
 ^^^^^^^^^^^
 
+- updated wcslib to v 6.2 [#8592]
```

Comment:
I think it would be worth mentioning the improvement that the WCS class can now be used to parse headers that contain time axes?

Also super minor, I'd fix the capitalization as ``Updated WCSLIB to v6.2``